packages/cli/package.json

```diff
@@ -45,7 +45,7 @@
     "preact": "^10.5.13",
     "preact-render-to-string": "^5.1.19",
     "preact-router": "^3.0.1",
-    "puppeteer": "^9.1.1",
+    "puppeteer": "^14.4.0",
```

Comment:
Older version had some issues with SWs and upgrading also allows for top-level await in the evals which is convenient.